tuf/api/metadata.py

```diff
@@ -444,9 +453,10 @@ class Signed(metaclass=abc.ABCMeta):
     instance attributes.*
 
     Args:
-        version: Metadata version number.
-        spec_version: Supported TUF specification version number.
-        expires: Metadata expiry date.
+        version: Metadata version number. Default is 1.
+        spec_version: Supported TUF specification version. Default is the
+            version currently supported by the library.
+        expires: Metadata expiry date. Default is current date and time.
```

Comment:
Is it worth mention that these are optional in the child classes only? I was confused for a split-second, because of the `Optional` type hint but no default value in the constructor signature (I understand now that [`Optional[X]` just means `X | None`](https://docs.python.org/3/library/typing.html#typing.Optional) regardless of it being an optional arg). What about something like...

```suggestion
        expires: Metadata expiry date. If None is passed the current date and time is assigned.
```
(same for `version` and `spec_version`)

Reply:
Maybe that would be better -- the small but important difference between Optional and the argument actually being optional is indeed annoying and hard to keep clear